identity-and-access-management/secret-manager/reference-content/secret-types-json-format.mdx

```diff
@@ -0,0 +1,52 @@
+---
+meta:
+  title: Understanding secret types
+  description: Explore the Secret Manager secret types such as credentials, SSH keys, certificates, and more. Learn how to manage sensitive data securely using the Secret Manager API.
+content:
+  h1: Understanding secret types
+  paragraph: Explore the Secret Manager secret types such as credentials, SSH keys, certificates, and more. Learn how to manage sensitive data securely using the Secret Manager API.
+tags: secret-types json secret-api
+dates:
+  validation: 2024-08-30
+  posted: 2024-08-30
+categories:
+  - identity-and-access-management
+---
+
+[Secret types](/identity-and-access-management/secret-manager/concepts/#secret-types) are the different types of sensitive data you can create when using Secret Manager.
+
+This documentation page provides information about the different secret types available with the Secret Manager API, and how to use them with Terraform.
+
+## Secret types available with the API
+
+You can create the other following secret types using Secret Manager:
+
+- **Opaque**: This is the default type, where the content is not restricted to any specific format. You can store any arbitrary data in an opaque secret.
+
+- **Basic credentials**: this secret type is a flat JSON that allows you to set a username and a password.
```

Comment:
capitalization after colons isn't consistent, I think it could be either one way or the other but should all be the same